std_msgs/Header header ##subscrive sensor-data time
sensor_msgs/PointCloud2 orig_cloud
sensor_msgs/PointCloud2 vox_cloud
sensor_msgs/PointCloud2 del_cloud
sensor_msgs/PointCloud2 clu_cloud
new_exploration_programs/index[] clu_indices
new_exploration_programs/f_vector[] clu_features
geometry_msgs/Point[] clu_centroids
